### PR TITLE
MAKE-782: Implement buildlogger routes 

### PR DESCRIPTION
**Pull request by julianedwards**

### CODE REVIEW

File: rest/parse.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package rest
```
Comment (tychoish) [inline rest/parse.go]: move this to util package

File: rest/parse.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import (
+	"net/url"
+	"time"
+
+	"github.com/evergreen-ci/cedar/util"
+	"github.com/pkg/errors"
+)
+
+func parseTimeRange(vals url.Values, start, end string) (util.TimeRange, error) {
+	tr := util.TimeRange{EndAt: time.Now()}
```
Comment (tychoish) [inline rest/parse.go]: why now?

File: rest/service.go [hunk 3]
```diff
@@ -230,4 +230,9 @@ func (s *Service) addRoutes() {
 	s.app.AddRoute("/perf/task_name/{task_name}").Version(1).Get().RouteHandler(makeGetPerfByTaskName(s.sc))
 	s.app.AddRoute("/perf/version/{version}").Version(1).Get().RouteHandler(makeGetPerfByVersion(s.sc))
 	s.app.AddRoute("/perf/children/{id}").Version(1).Get().RouteHandler(makeGetPerfChildren(s.sc))
+
+	s.app.AddRoute("/log/{id}").Version(1).Get().RouteHandler(makeGetLogByID(s.sc))
+	s.app.AddRoute("/log/meta/{id}").Version(1).Get().RouteHandler(makeGetLogMetaByID(s.sc))
+	s.app.AddRoute("/log/task_id/{task_id}").Version(1).Get().RouteHandler(makeGetLogByTaskID(s.sc))
+	s.app.AddRoute("/log/meta/task_id/{task_id}").Version(1).Get().RouteHandler(makeGetLogMetaByTaskID(s.sc))
```
Comment (tychoish) [inline rest/service.go]: buildlogger

File: rest/parse.go (hunk 2 above)
Comment (julianedwards) [reply rest/parse.go]: Default time range is 0 time to current time. Line 29 below changes it if endAt is not empty.

File: rest/parse.go (hunk 1 above)
Comment (julianedwards) [reply rest/parse.go]: Even though this is specific to URL parsing?

**Pull request merged**